demos/apps/sleep/sleep.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
Does tutum/curl listen on any ports? If not, it might be confusing to create a k8s service for it. Alternatively, you could use gcr.io/google_containers/echoserver:1.4 image which has a basic echo server (port 8080) and also includes curl for client testing.